requirements/requirements_tts.txt

```diff
@@ -4,7 +4,8 @@ jieba
 kornia
 librosa
 matplotlib
-nemo_text_processing
+# pynini does not currently support aarch, disable nemo_text_processing for now
+nemo_text_processing; 'arm' not in platform_machine and 'aarch' not in platform_machine
```

Comment:
should we move this up to requirements or leave it in tts? I see it's used in

- examples/nlp
- NeMo/nemo/collections/asr/data/text_to_text.py
- NeMo/nemo/collections/common/tokenizers/text_to_speech/ipa_lexicon.py
- NeMo/nemo/collections/nlp/models/machine_translation/mt_enc_dec_model.py

and others. This is not a complete list

Reply:
Lets guard all of this at their init and remove nemo text processing install entirely. Unless you are sure this `'arm' not in platform_machine and 'aarch' not in platform_machine` thing actually works?

Reply:
I can push the changes to the rest of the files if you want.

Reply:
I would prefer to keep the nemo text processing install.

Reply:
We should either remove it, or test that the arm and aarch stuff works. If it works, ok to move it to commons, but its fine if its in tts too since all the text processing accesses are guarded inside nemo/collections